tests/API/axios.spec.ts

```diff
@@ -0,0 +1,85 @@
+import axios, { Axios } from 'axios';
+import { wrapper } from 'axios-cookiejar-support';
+import test, { expect } from 'playwright/test';
+import { CookieJar } from 'tough-cookie';
+
+test.describe.only('Mocking response', async () => {
+     let client: Axios;
+     let userID: string;
+     let token: string;
+    
+     
+  test.beforeAll(async ({}) => {
+    const jar = new CookieJar();
+   
+    client = wrapper(axios.create({
+      baseURL: process.env.API_URL,
+      validateStatus: function (status) {
+        return status >= 200 && status < 300; 
+      },
+      jar
+    }));
+
+    const loginData = await client.post('Account/v1/Login', {
+      "userName": "MaryI",
+      "password": "Test123!"
```

Comment:
use process.env.USERNAME
```suggestion
      "userName": process.env.USERNAME
      "password": process.env.PASSWORD
```